bench/src/main/scala/sjsonnet/OptimizerBenchmark.scala

```diff
@@ -3,6 +3,8 @@ package sjsonnet
 import java.io.StringWriter
 import java.util.concurrent.TimeUnit
 
+import scala.collection.mutable.HashMap
```

Comment:
I'd prefer to keep mutable types qualified (i.e. only import `scala.collection.mutable`) everywhere for consistency.